Dockerfile

```diff
@@ -1,23 +1,29 @@
-FROM python:3.9-slim as build
+FROM node:lts-buster as node-build
+WORKDIR /root
+COPY . .
+RUN yarn install && make assets-build
+
 
-RUN apt-get update && apt-get install -y g++ gcc libxslt-dev make libcurl4-openssl-dev build-essential
 
+FROM python:3.9-slim as build
+RUN apt-get update && apt-get install -y g++ gcc libxslt-dev make libcurl4-openssl-dev build-essential
 RUN apt-get install -y libssl-dev
+WORKDIR /root
 
-RUN mkdir -p /var/www/scylla
-WORKDIR /var/www/scylla
+COPY . .
+COPY --from=node-build /root/scylla/assets ./scylla
 
-RUN pip install scylla
+RUN pip install -r requirements.txt -i https://mirrors.aliyun.com/pypi/simple/  && python setup.py install
```

Comment:
We don't need this because GH Actions are running overseas.

I will find a way to fix the CI.

Reply:
The CI has been fixed.